doc/pilight-uuid.1

```diff
@@ -0,0 +1,46 @@
+.TH pilight-uuid 1 "3 December 2016" "7.0" "pilight 7.0"
+.SH NAME
+pilight-uuid \- Generate pilight adhoc node UUID
```

Comment:
This does not `generate` an UUID, but prints it. The UUID is the same all the time as long as the MAC address does not change.